src/components/Header/Header.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, text, number } from "@storybook/addon-knobs";
+import Header from './index';
+
+
+storiesOf('Text', module)
+	.addDecorator(
+		withInfo({
+			Heading: "This is a header component"
+		})
+	)
+	.addDecorator(
+		withKnobs() 
+	)
+	.add('Header', () => (
+    <Header 
+    heading={text('text', 'Sample text')} 
+    size={number('heading-level', '2')} 
```

Comment:
Can we keep the argument consistent with the props name? Let's use **size** instead of **heading-level**
Something like `size={number('size', '2')`